docs/webiny-apps/page-builder/development/creating-iframe-element-plugin.md

```diff
@@ -31,29 +31,36 @@ Head over to our [Webiny forum](https://community.webiny.com/), [Gitter](https:/
 
 ### Editor
 
-One of the main sections of the Page Builder App is the Editor, you can get there by editing a site that you create. By default, it offers you a variety of elements or plugins as seen in the image below.
+One of the main sections of the Page Builder App is the Editor, you can open the Editor by editing the page that you create.
+Out-of-the-box the PB editor includes a myriad of page elements.
 
 ![Editor Elements](/img/webiny-apps/page-builder/development/development/plugin-reference/editor/iframe/editor-elements.png)
 
 
-When building pages, you are using these elements (e.g. Text, Image, Form, ...), which are registered in the system via plugins. In terms of rendering, every element has to have two plugins, one that defines how it's rendered in the actual page builder, and the other that defines how it's rendered on the website.
+This list of page elements can be expanded and custom page elements can be created via plugins. In order to create a new page element, we need to register two plugins, one that defines how it's rendered in the actual page builder, and the other that defines how it's rendered on the website.
 
 If you are interested on different types of plugins, each app in Webiny consistes of the `types.ts` that includes all the plugin types of that app.
 In our case is [app-page-builder/src/types.ts](https://github.com/webiny/webiny-js/blob/master/packages/app-page-builder/src/types.ts)
 
+### Editor Plugin
+
 Let's render our plugin in the page builder, first we will use the `PbEditorPageElementPlugin` plugin type
```

Comment:
![image](https://user-images.githubusercontent.com/5121148/81854309-0d403e80-955e-11ea-8936-c2ece30ec2c1.png)

1. A couple of things for the first sentence (`Let's render our plugin in the page builder, ...`).
A) We are not rendering a plugin, we are rendering a page element.
B) `in the page builder` - actually, it's in the "editor", which is the word we use in the article to reference the actual page editor
C) Please put a dot at the end, instead of the comma.

2. Please use `pb-editor-page-element` instead of the actual type name, and of course, preserve the link to the reference. This is how we've done it in other articles, so this is just for consistency sake.

3. Missing dot.

4. Remove this please.

5. The first line in the code is blank.

6. please put a blank line in between 

7. please put a blank line in between

8. no need to return an array, a plain object is just fine.

9. please check the ending of the code snippet (missing some parentheses I think), this is not marked on sshot

10. please add the import statement for the `styled` as well (`import styled from "@emotion/styled";`). We want the users to know from where these imported functions are coming. Apply the same for other snippets as well.